src/main/java/practice/StreamPractice.java

```diff
@@ -13,8 +16,14 @@ public class StreamPractice {
      * If there is no needed data throw RuntimeException with message
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
-    public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+    public static int findMinEvenNumber(List<String> numbers) {
+        return numbers.stream()
+                .flatMap(str -> Arrays.stream(str.split(",")))
```

Comment:
For the future, do not use abbreviations
```suggestion
                .flatMap(lineNumbers -> Arrays.stream(lineNumbers.split(",")))
```